<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Movie Download</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>20</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="placeholderText">
      <string>Movie name</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>150</y>
       <width>551</width>
       <height>23</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>20</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
   text-shadow: 2px 2px 5px red;
}</string>
     </property>
     <property name="text">
      <string>Check Movie</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="currentText">
      <string>Choose your movie</string>
     </property>
     <item>
      <property name="text">
       <string>Choose your movie</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>80</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Location</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Quality">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Choose you quality</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="save_location_button">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>80</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Save Location</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_download">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Checkquality">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Check Quality</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_downloadspeed">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>200</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
</string>
     </property>
     <property name="text">
      <string>Download Speed</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_downloadspeedcurrent">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>200</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: Black;
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_downloadspeed_2">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>200</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
</string>
     </property>
     <property name="text">
      <string>Remaining</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Remaining">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>200</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: Black;
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Subtitles</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="QRC Files/Test.qrc"/>
 </resources>
 <connections/>
</ui>
